lib/widgets/chiplist.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+class Genres extends StatelessWidget {
+  Genres({super.key, required this.genres});
+
+  List<String> genres;
+  static const double labelPaddingEdgeInsets = 2.0;
+  static const double spacingWrap = 6.0;
+  static const double runSpacingWrap = 6.0;
+  static const double paddingEdgeInsets = 8.0;
+
+  Widget _buildChip(String label) {
+    return Chip(
+      labelPadding: const EdgeInsets.all(labelPaddingEdgeInsets),
+      avatar: CircleAvatar(
+        backgroundColor: Colors.black,
+        child: Text(label[0].toUpperCase(),),
+      ),
+      label: Text(
+        label,
+        style: const TextStyle(
+          color: Colors.white,
+          fontStyle: FontStyle.italic,
+        ),
+      ),
+      backgroundColor: Colors.red[300],
```

Comment:
Instead to call it like Colors.red[300] you can change it to Colores.red.shade300

Reply:
solved